.github/workflows/cbmc-update.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+name: Attempt CBMC update
+
+on:
+  schedule:
+    - cron: "30 4 * * Mon" # Run this every Monday at 04:30 UTC
+  workflow_dispatch:     # Allow manual dispatching for a custom branch / tag.
+
+permissions:
+  checks: write
+  contents: write
+  issues: write
+  pull-requests: write
+
+jobs:
+  create-cargo-update-pr:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Kani
+        uses: actions/checkout@v3
+
+      - name: Setup Kani Dependencies
+        uses: ./.github/actions/setup
+        with:
+          os: ubuntu-22.04
+
+      - name: Compare CBMC versions
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
```

Comment:
This step doesn't only compare CBMC versions. I suggest splitting this step into smaller ones where you (1) get the current CBMC versions and log them, (2) get the latest CBMC versions and log them, (3) compare the CBMC versions, etc.

Reply:
I have updated the title of this step, but I am not going to split it up: doing so would create mini-steps for the first line, the second line, and then all the rest.